### PR TITLE
Add status page and table

### PR DESCRIPTION
**Pull request by brendanheywood**
### Pull request checks
- [x] I have checked the version numbers are correct as per the [README](./README.md#branches)

### CODE REVIEW

File: classes/table/exemption_table.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+            $row = [];
+            $row[] = $output->check_result($result);
+            $row[] = $output->action_link($link, $check->get_name());
+
+            $row[] = $result->get_summary()
+                    . '<br>'
+                    . \html_writer::start_tag('small')
+                    . $output->action_link($link, get_string('moreinfo'))
+                    . \html_writer::end_tag('small');
```
Comment (bwalkerl) [inline classes/table/exemption_table.php:84]: There's a couple of potential indenting issues here which were fixed in the following branch. Nothing serious so will depend on if you want to update it now or just wait until the later branch is merged in.

**Review by bwalkerl — COMMENTED** [review]
Two small things I noticed. Aside from those the page seems to mirror everything else nicely as intended.

File: status.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+define('NO_OUTPUT_BUFFERING', true);
+
+require('../../../config.php');
+require_once($CFG->libdir.'/adminlib.php');
+
+admin_externalpage_setup('reportstatus', '', null, '', ['pagelayout' => 'report']);
+
+$detail = optional_param('detail', '', PARAM_TEXT); // Show detailed info about one check only.
+
+$url = '/report/status/status.php';
```
[the remark this answers is not in the document]
Comment (brendanheywood) [reply status.php:34]: this should be admin/tool/heartbeat/status.php

[the remark this answers is not in the document]
Comment (bwalkerl) [reply status.php:34]: Tested that and it works, so +1 for changing it to `/admin/tool/heartbeat/status.php`

**Comment by brendanheywood** [comment]
closing this in favor of 171

**Pull request closed without merging**